libnopegl/src/backends/vk/gpu_ctx_vk.c

```diff
@@ -1054,7 +1059,7 @@ static int vk_begin_draw(struct gpu_ctx *s, double t)
     s_priv->cur_cmd = s_priv->cmds[s_priv->cur_frame_index];
     VkResult res = ngli_cmd_vk_begin(s_priv->cur_cmd);
     if (res != VK_SUCCESS)
-        return res;
+        return ngli_vk_res2ret(res);
```

Comment:
the `ngli_cmd_vk_add_wait_sem` ret code below is also a `VkResult` (you adjusted that but in the wrong commit)

Reply:
Done.